app/views/foods/_food.html.erb

```diff
@@ -20,5 +18,3 @@
     <strong>Quantity:</strong>
     <%= food.quantity %>
```

Comment:
- [ ] [REQUIRED] Good job so far! Has an index page for the food list that matches the given [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/images/food_index.png?raw=true).
and has a `add` new food item `button` that leads to the form that allows users to `add` new food
and also can `delete` a food item from the lists however I couldn't implement these functions because I couldn't find them. Please solve this problem 🙏🏼 

![image](https://user-images.githubusercontent.com/63915024/209191655-4cf3c1d5-d8cd-49b9-8903-44edfa89ea3e.png)